crates/hyperqueue/src/client/commands/log.rs

```diff
@@ -53,6 +60,9 @@ pub enum LogCommand {
 
     /// Prints a raw content of one channel
     Cat(CatOpts),
+
+    /// Export log into JSON
+    Export(ExportOpts),
```

Comment:
Maybe it would be better if we just outputted JSON when the user combines `cat` with `--output-mode=json`?